Makefile

```diff
@@ -653,10 +654,6 @@ ${PROOFTHEORIES}: %_proofs.thy: %.pbf ${BF_GEN_PATH} ${STATICSOURCES} \
 # Preprocessed source files
 ###########################
 
-%.s: %.S ${GENHEADERS} ${STATICHEADERS} | ${DIRECTORIES}
-	@echo " [CPP] $@"
-	$(Q)${CPP} ${CPPFLAGS} -CC -E -o $@ $<
-
```

Comment:
```
%.o: %.S | ${DIRECTORIES}
        @echo " [AS] $@"
        $(Q)${CC} ${ASFLAGS} -c $< -o $@
```

if using gcc compiling .S file,It's will throught C pre-processor.
or
change .s to .S.s or something that don't overwrite the .S source code.